src/Symfony/Component/Validator/Mapping/Deprecated/TraversalStrategy.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Mapping\Deprecated;
+
+trigger_error('Constants STOP_RECURSION in class Symfony\Component\Validator\Mapping\TraversalStrategy is deprecated since version 2.3 and will be removed in 3.0.', E_USER_DEPRECATED);
```

Comment:
We cannot remove these usages AFAIK given that they are needed to provide the BC layer.

But IIRC, this constant has been *added* to be able to provide a BC layer, so it could be marked with ``@internal`` (in the 2.5 branch)